src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb

```diff
@@ -1682,5 +1682,28 @@ class C
             End Using
 
         End Function
+
+        <WpfFact, Trait(Traits.Feature, Traits.Features.Completion)>
+        Public Async Function TestRecursiveGenericSymbolKey() As Task
```

Comment:
I agree.  It was a brainfart that made me forget to do a symbolkey test.  But i do like an actual test that goes through the full codepath.